homeassistant/components/opnsense/strings.json

```
@@ -0,0 +1,27 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "OPNsence",
+        "description": "Set required parameters to connect to your router. For more information, please refer to the [integration documentation]({doc_url})",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "port": "[%key:common::config_flow::data::port%]",
+          "api_key": "[%key:common::config_flow::data::api_key%]",
+          "api_secret": "API Secret",
```

This needs sentence-casing:

```suggestion
          "api_secret": "API secret",
```